bundles/plugins/org.bonitasoft.studio.properties/src/org/bonitasoft/studio/properties/sections/subprocess/ContractInputNamePredicate.java

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
You may want to reuse ContractInputPredicates.withContractInputName(String) in order to syndicate all this ;)